src/components/listBox/ListBoxConnected.tsx

```diff
@@ -15,12 +15,12 @@ const mapStateToProps = (state: IReactVaporState, ownProps: IListBoxOwnProps): I
     return {
         selected: list ? list.selected : [],
         active: list ? list.active : undefined,
-        items: list ? list.items : ownProps.items,
+        items: ownProps.items || [],

```

Comment:
You can remove this whole line. No need to redefine items' value in mapStateToProps if you don't take any value from the state.